editor/project_manager.cpp

```diff
@@ -2158,8 +2158,9 @@ void ProjectManager::_run_project() {
 }
 
 void ProjectManager::_scan_dir(const String &path, List<String> *r_projects) {
-	DirAccess *da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
-	da->change_dir(path);
+	DirAccessRef da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+	Error error = da->change_dir(path);
+	ERR_FAIL_COND_MSG(error != Error::OK, "Could not scan directory " + path);
```

Comment:
We usually use error code without specifying the enum:
```suggestion
	ERR_FAIL_COND_MSG(error != OK, "Could not scan directory at: " + path);
```